app/assets/stylesheets/application/components/ann-work-profile-light.scss

```diff
@@ -76,5 +76,29 @@
         }
       }
     }
+
+    .prev-episode, .next-episode {
+      background-color: rgba(224, 86, 118, 0.8);
+      color: #ffffff;
+      font-size: 12px;
```

Comment:
Properties should be ordered background-color, color, display, font-size, padding, position, top